<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>367</width>
    <height>342</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Atmospheric correction</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Satellite</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>DN file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_DN">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Add path to satellite image with digital numbers</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="toolButton_DN">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Metafile</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_meta">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Add path to metadata file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QToolButton" name="toolButton_meta">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_ispan">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Panchromatic</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Method</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_atmParameters">
       <property name="text">
        <string>Atmospheric parameters:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_atmProfile">
       <property name="text">
        <string>Atmospheric profile</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox_modis">
       <property name="text">
        <string>Download from MODIS</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_AOT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>AOT</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_aot">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Atmospherical optical thickness</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_PWV">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>PWV</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_pwv">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Precipitable water vapor</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_Ozone">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ozone</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_ozone">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Ozone concentration</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_output">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Add path to output file (.tif)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QToolButton" name="toolButton_output">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_save">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox_atmProfile">
       <item>
        <property name="text">
         <string>No aerosols</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Continental</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maritime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Urban</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Desert</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biomass burning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stratospheric</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox_method">
       <item>
        <property name="text">
         <string>DOS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RAD</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox_satellite">
       <item>
        <property name="text">
         <string>WorldView-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WorldView-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Landsat-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Landsat-7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pleiades-A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pleiades-B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SPOT-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sentinel-2A, 10m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sentinel-2A, 60m</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_tileSize">
       <property name="text">
        <string>Tile size (pixels)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSpinBox" name="spinBox_tileSize">
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
